frontend/packages/knative-plugin/src/actions/add-subscription.tsx

```diff
@@ -0,0 +1,17 @@
+import { KebabOption } from '@console/internal/components/utils';
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { EventingSubscriptionModel } from '../models';
+
+export const addSubscription = (model: K8sKind, source: K8sResourceKind): KebabOption => {
+  return {
+    label: 'Add Subscription',
+    href: `add/ns/${source.metadata.namespace}`,
+    accessReview: {
+      group: EventingSubscriptionModel.apiGroup,
+      resource: EventingSubscriptionModel.plural,
+      name: source.metadata.name,
+      namespace: source.metadata.namespace,
+      verb: 'create',
+    },
+  };
```

Comment:
can we have a common action here `add-pubsub,tsx` instead of having separate for subscription and trigger, WDYT?

 ```suggestion
  return {
    label: `Add ${model.kind}`,
    href: `add/ns/${source.metadata.namespace}`,
    accessReview: {
      group: model.apiGroup,
      resource: model.plural,
      name: source.metadata.name,
      namespace: source.metadata.namespace,
      verb: 'create',
    },
  };
```

Reply:
Added this way, because each of this actions will have a different callbacks and will be easy to maintain